tests/test_diaCalculationPlugins.py

```diff
@@ -872,6 +880,76 @@ def testCalculate(self):
                                np.sqrt(1 / (n_sources - 1)))
 
 
+class TestMultiLombScarglePeriodogram(unittest.TestCase):
+
+    def testCalculate(self):
+        """Test Mulitband Lomb Scargle Periodogram."""
+        n_sources = 10
+        objId = 0
+
+        # Create synthetic multi-band data
+        times = np.linspace(0, 2*np.pi, n_sources)
+        fluxes = np.sin(0.3 * times)
+        diaObjects = pd.DataFrame({"diaObjectId": [objId]})
+        diaSources = pd.DataFrame(
+            data={"diaObjectId": n_sources * [objId],
+                  "band": n_sources * ["u"],
+                  "diaSourceId": np.arange(n_sources, dtype=int),
+                  "midpointMjdTai": times,
+                  "psfFlux": fluxes,
+                  "psfFluxErr": np.zeros(n_sources)})
+
+        plug = LombScarglePeriodogramMulti(LombScarglePeriodogramMultiConfig(),
+                                           "ap_lombScarglePeriodogramMulti",
+                                           None)
+
+        run_multi_plugin(diaObjects, diaSources, "u", plug)
+        self.assertAlmostEqual(diaObjects.at[objId, "multi_period"],
+                               31.41592653589793)
```

Comment:
Where did this value come from? It's 10*pi, but that being the answer is not obvious to someone unfamiliar with the algorithm.